src/OrchardCore.Modules/OrchardCore.Templates/Views/Template/Index.cshtml

```diff
@@ -5,7 +5,7 @@
     int endIndex = startIndex + Model.Templates.Count - 1;
 }
 
-<form asp-action="Index">
+<form asp-controller="Template" method="get">
```

Comment:
A method by default GET, szóval nem kell megadni, nem?

Reply:
Igen, ki is vettem